onnxruntime/core/providers/cpu/ml/tree_ensemble_helper.h

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT License.
 
 #pragma once
+
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
TreeEnsembleClassifier and TreeEnsembleRegressor are ML ops that could be validly included in a minimal build. 

@xadupre is this usage optional and calling GetVectorAttrsOrDefault is an optimization that we can't do in a minimal build?

https://github.com/microsoft/onnxruntime/blob/3fb8905393bf91d97eae7afa0ae341828e3f1dbc/onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h#L101-L106

Reply:
If we need to fix this, let's do it in another PR.